src/controller/python/ChipDeviceController-ScriptBinding.cpp

```diff
@@ -557,38 +611,6 @@ ChipError::StorageType pychip_ScriptDevicePairingDelegate_SetCommissioningStatus
     return CHIP_NO_ERROR.AsInteger();
 }
 
```

Comment:
There is no reason why platform::MemoryInit would be somewhere else than a global init - it has nothing to do with devicecontroller.